File: 2026/06-June/18-Thursday/solution.kt
class Solution {
    fun angleClock(hour: Int, minutes: Int): Double {
        val hourAngle = (hour % 12) * 30.0 + minutes * 0.5
        val minuteAngle = minutes * 6.0

        val diff = kotlin.math.abs(hourAngle - minuteAngle)

        return minOf(diff, 360.0 - diff)
    }
}

File: 2026/06-June/18-Thursday/solution.php
class Solution {

    /**
     * @param Integer $hour
     * @param Integer $minutes
     * @return Float
     */
    function angleClock($hour, $minutes) {
        $hourAngle = ($hour % 12) * 30 + $minutes * 0.5;
        $minuteAngle = $minutes * 6;

        $diff = abs($hourAngle - $minuteAngle);

        return min($diff, 360 - $diff);
    }
}
